server/apiRoutes/post.js

```diff
@@ -44,7 +44,21 @@ router.get('/groupPosts/:groupId', async (req, res, next) => {
     }
 });
 
-  // grab a single post i.e. when someone clicks on a post link to read comments
+//get all posts associated to an event
+router.get('/eventPosts/:eventId', async (req, res, next) => {
```

Comment:
I don't think events will have posts. @katherinepeterson do events have posts?

Reply:
Ahh yeah, I forgot that they're a type of post.

Reply:
I shouldn't need to use the switch inside of the loadPosts thunk then correct?